### ISSUE TITLE
twisted.test.test_postfix fails on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8713]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8713|
|Type|enhancement|
|Created|2016-08-01 08:36:03Z|
|Branch|https://github.com/twisted/twisted/tree/8713-rodrigc-postfix|

twisted.test.test_postfix fails on Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8713 8713
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8713_rodrigc_postfix 8713-rodrigc-postfix
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470040563311623 1470040563311623
changetime__1470055100583230 1470055100583230
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/450

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

changes looks good. Please merge

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset cfdd18b2f1fa45b1901623cb46fbfb46be4524dc
```
#!CommitTicketReference repository="" revision="cfdd18b2f1fa45b1901623cb46fbfb46be4524dc"
Merge pull request #450 from twisted/8713-rodrigc-postfix

Author: rodrigc
Reviewer:  acabhishek942
Fixes: #8713

Convert twisted.protocols.postfix to Python 3
```
